acquia_lift.module

```diff
@@ -2853,35 +2778,16 @@ function acquia_lift_command_messagebox($message, $seconds = 0) {
 }
 
 /**
- * Returns an AJAX command to toggle the page variation element selector.
+ * Returns an AJAX command to toggle the element variation element selector.
  *
  * @param bool $start
- *   True to start the page variation mode, FALSE to stop it.
+ *   True to start the element variation mode, FALSE to stop it.
  */
-function acquia_lift_command_page_variation_toggle($start = TRUE) {
+function acquia_lift_command_element_variation_toggle($start = TRUE) {
   return array(
     'command' => 'acquia_lift_variation_toggle',
     'data' => array(
       'start' => $start,
-      'type' => 'page',
-    ),
-  );
-}
-
-/**
- * Returns an AJAX command to preview a specific page variation.
- *
- * @param string $agent_name
- *   The name of the campaign/agent to which the variation belongs.
- * @param int $variation_index
- *   The index of the variation to preview.
- */
-function acquia_lift_command_page_variation_preview($agent_name, $variation_index) {
```

Comment:
We should still have the ability to indicate that we want to preview a specific variation... so yes - this functionality should stay.  I'm not sure why you aren't getting errors.  I see what you mean about the old variation preview JS command... that is definitely not what we keep.  The answer may lie here: https://github.com/acquia/acquia_lift/blob/CH-2304/src/js/menu/acquia_lift.personalize.commands.js#L6